config/protocol.mainnet.neofs.yml

```diff
@@ -26,6 +26,7 @@ ProtocolConfiguration:
   Hardforks:
     Aspidochelone: 3000000
     Basilisk: 4500000
+    Cockatrice: TODO
```

Comment:
@roman-khimov, we need to decide about this and testnet configs, and after it I'll update the https://github.com/neo-project/neo/blob/master/src/Neo.CLI/config.fs.mainnet.json.

Reply:
5800000, ~two weeks from now.

Reply:
Fixed.